transmart-data/test_data/studies/SURVEYS/i2b2demodata/study.tsv

```diff
@@ -40,7 +40,8 @@
          ""type"":""String"",
          ""width"":400,
          ""columns"":400,
-         ""measure"":""Nominal""
+         ""measure"":""Nominal"",
+									""missingValues"": {""values"":[""NA""]}
```

Comment:
lots of whitespaces

Reply:
fixed. please see the "File changed" tab for the refreshed content.